op-devstack/dsl/l2_cl.go

```diff
@@ -44,3 +53,72 @@ func (cl *L2CLNode) Start() {
 func (cl *L2CLNode) Stop() {
 	cl.control.L2CLNodeState(cl.inner.ID(), stack.Stop)
 }
+
+func (cl *L2CLNode) SyncStatus() *eth.SyncStatus {
+	ctx, cancel := context.WithTimeout(cl.ctx, DefaultTimeout)
+	defer cancel()
+	syncStatus, err := cl.inner.RollupAPI().SyncStatus(ctx)
+	cl.require.NoError(err)
+	return syncStatus
+}
+
+func (cl *L2CLNode) HeadBlockRef(label string) eth.BlockRef {
+	syncStatus := cl.SyncStatus()
+	targetWrap := reflect.ValueOf(syncStatus).Elem().FieldByName(label)
+	cl.require.True(targetWrap.IsValid(), "invalid label")
+	target := targetWrap.Interface()
+	L2Block, ok := target.(eth.L2BlockRef)
+	if ok {
+		return L2Block.BlockRef()
+	}
+	L1Block, ok := target.(eth.L1BlockRef)
+	cl.require.True(ok, "invalid type")
+	return L1Block
+}
+
+func (cl *L2CLNode) ChainID() eth.ChainID {
+	return cl.chainID
+}
+
+func (cl *L2CLNode) Advance(label string, delta uint64, attempts int) CheckFunc {
```

Comment:
nit: Advance sounds like it's causing the chain to advance rather than waiting for it to advance.